### PR TITLE
Change all instances of "Focussed" to "Focused"

### LINKED ISSUE #3165: Focussed and Focused
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

Path of Exile is using the term "Focused", here is an example:

`{crafted}Your Critical Strike Chance is Lucky while Focused`

I should be able to select the option "Are you Focused?" in the Configuration tab.

### What is the actual behaviour/value?

Path of Building however does not detect the term "Focused" and expects it to be "Focussed", which after changing that is accepted in Path of Building and then I'm able to select the option "Are you Focussed?" in the Configuration tab.

So there are two typos to be fixed in Path of Building, one the detection of the mod and two the typo on the Configuration page.

### How to reproduce the issue

_No response_

### Build code

```shell
None
```

### Screenshots

image
image

### PR DESCRIPTION
**Pull request by LocalIdentity**
GGG changed the wording of the Focused mod so I've also updated it's spelling in out internal variable names too. Only place that I didn't change the naming was in the patch notes
Fixes #3165

### CODE REVIEW

**Review by ppoelzl — APPROVED** [review]
LGTM.

File: src/Modules/ModParser.lua [hunk 1]
```diff
@@ -1165,7 +1165,7 @@ local modTagList = {
 	["while in sand stance"] = { tag = { type = "Condition", var = "SandStance" } },
 	["while you have a bestial minion"] = { tag = { type = "Condition", var = "HaveBestialMinion" } },
 	["while you have infusion"] = { tag = { type = "Condition", var = "InfusionActive" } },
-	["while focussed"] = { tag = { type = "Condition", var = "Focused" } },
+	["while focus?sed"] = { tag = { type = "Condition", var = "Focused" } },
```
Comment (maximumstock) [inline src/Modules/ModParser.lua:1168]: Hey, I'm trying to find my way around the codebase and was wondering about this change. What is the reason for this `?` notation?

Comment (zao) [reply src/Modules/ModParser.lua:1168]: The string on the left hand side is a Lua _pattern_ where the `?` denotes that the character/class preceding it is optional. This construct allows our code to keep matching both the old and new spelling. See <https://www.lua.org/pil/20.2.html> for info on patterns.

Comment (maximumstock) [reply src/Modules/ModParser.lua:1168]: @zao Awesome, thank you very much!

**Pull request merged**